public/components/report_definitions/report_trigger/report_trigger.tsx

```diff
@@ -76,10 +101,25 @@ export function ReportTrigger() {
 
   const handleReportTriggerType = (e: React.SetStateAction<string>) => {
     setReportTriggerTypes(e);
+    if (e === 'scheduleOption') {
+      trigger_schema['trigger_type'] = 'Schedule';
+    } else if (e === 'alertOption') {
+      trigger_schema['trigger_type'] = 'Alert';
+    }
```

Comment:
refer to below comment, use map to simplify code and improve readibility

Reply:
Will change